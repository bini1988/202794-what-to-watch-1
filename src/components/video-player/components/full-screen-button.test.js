import React from "react";
import renderer from "react-test-renderer";
import FullScreenButton from "./full-screen-button";

it(`FullScreenButton correctly renders default markup`, () => {
  const tree = renderer.create(
      <FullScreenButton onClick={() => {}}/>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
