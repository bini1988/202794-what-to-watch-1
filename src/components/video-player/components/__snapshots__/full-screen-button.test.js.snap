// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FullScreenButton correctly renders default markup 1`] = `
<button
  className="player__full-screen"
  onClick={[Function]}
  type="button"
>
  <svg
    height="27"
    viewBox="0 0 27 27"
    width="27"
  >
    <use
      xlinkHref="#full-screen"
    />
  </svg>
  <span>
    Full screen
  </span>
</button>
`;
